src/context/PageContext.js

```diff
@@ -0,0 +1,7 @@
+import { createContext } from 'react'
+
+export const HomePageContext = createContext(/* boolean */ false)
+export const ModalPageContext = createContext(/* boolean */ false)
+export const LogoStyleContext = createContext(
+  /* 'default' | 'white' */ 'default'
```

Comment:
I really want TS here ^^

Reply:
Yah this would be a much safer way of doing things...